docs/VARIABLES.md

```diff
@@ -2548,9 +2548,23 @@ Default:  none
 
 ***
 
+<<<<<<< HEAD
```

Comment:
merge conflict chars

Reply:
Strange. It doesnt show this to me locally. Will check and fix it